bitsandbytes/autograd/_functions.py

```diff
@@ -106,7 +106,14 @@ def undo_layout(permuted_tensor: torch.Tensor, tile_indices: torch.LongTensor) -
 
 class MatMul8bit(torch.autograd.Function):
     @staticmethod
-    def forward(ctx, A, B, out=None, quant_type="vector", precision=None):
+    def forward(
+        ctx,
+        A: torch.Tensor,
+        B: torch.Tensor,
+        out=None,
+        quant_type: F.VectorwiseQuantType = "vector",
+        precision=Optional[List[int]],
```

Comment:
⚠️ Type annotation had become the default value here?
```suggestion
        precision: Optional[List[int]] = None,
```

Reply:
Good catch, thanks!